src/controllers/post.controller.ts

```diff
@@ -42,9 +43,9 @@ export const renderPostForm = asyncHandler(async (req: Request, res: Response) =
 export const getPostsForGuest = asyncHandler(async (req: Request, res: Response) => {
   const currentUserId = req.session.user?.id;
   if (currentUserId) {
-    res.redirect(`/posts/fyp/1`);
+    res.redirect(`/posts/fyp?page=1`);
```

Comment:
mặc định sẽ là page = 1 rồi nên chỗ này ko cần truyền thêm đâu nha

```suggestion
    res.redirect(`/posts/fyp`);
```